crates/vm/levm/src/opcodes.rs

```diff
@@ -170,161 +170,164 @@ pub enum Opcode {
-            0x69 => Opcode::PUSH10,
-            0x6A => Opcode::PUSH11,
-            0x6B => Opcode::PUSH12,
-            0x6C => Opcode::PUSH13,
-            0x6D => Opcode::PUSH14,
-            0x6E => Opcode::PUSH15,
-            0x6F => Opcode::PUSH16,
-            0x70 => Opcode::PUSH17,
-            0x71 => Opcode::PUSH18,
-            0x72 => Opcode::PUSH19,
-            0x73 => Opcode::PUSH20,
-            0x74 => Opcode::PUSH21,
-            0x75 => Opcode::PUSH22,
-            0x76 => Opcode::PUSH23,
-            0x77 => Opcode::PUSH24,
-            0x78 => Opcode::PUSH25,
-            0x79 => Opcode::PUSH26,
-            0x7A => Opcode::PUSH27,
-            0x7B => Opcode::PUSH28,
-            0x7C => Opcode::PUSH29,
-            0x7D => Opcode::PUSH30,
-            0x7E => Opcode::PUSH31,
-            0x7F => Opcode::PUSH32,
-            0x80 => Opcode::DUP1,
-            0x81 => Opcode::DUP2,
-            0x82 => Opcode::DUP3,
-            0x83 => Opcode::DUP4,
-            0x84 => Opcode::DUP5,
-            0x85 => Opcode::DUP6,
-            0x86 => Opcode::DUP7,
-            0x87 => Opcode::DUP8,
-            0x88 => Opcode::DUP9,
-            0x89 => Opcode::DUP10,
-            0x8A => Opcode::DUP11,
-            0x8B => Opcode::DUP12,
-            0x8C => Opcode::DUP13,
-            0x8D => Opcode::DUP14,
-            0x8E => Opcode::DUP15,
-            0x8F => Opcode::DUP16,
-            0x90 => Opcode::SWAP1,
-            0x91 => Opcode::SWAP2,
-            0x92 => Opcode::SWAP3,
-            0x93 => Opcode::SWAP4,
-            0x94 => Opcode::SWAP5,
-            0x95 => Opcode::SWAP6,
-            0x96 => Opcode::SWAP7,
-            0x97 => Opcode::SWAP8,
-            0x98 => Opcode::SWAP9,
-            0x99 => Opcode::SWAP10,
-            0x9A => Opcode::SWAP11,
-            0x9B => Opcode::SWAP12,
-            0x9C => Opcode::SWAP13,
-            0x9D => Opcode::SWAP14,
-            0x9E => Opcode::SWAP15,
-            0x9F => Opcode::SWAP16,
-            0xA0 => Opcode::LOG0,
-            0xA1 => Opcode::LOG1,
-            0xA2 => Opcode::LOG2,
-            0xA3 => Opcode::LOG3,
-            0xA4 => Opcode::LOG4,
-            0x51 => Opcode::MLOAD,
-            0x52 => Opcode::MSTORE,
-            0x53 => Opcode::MSTORE8,
-            0x54 => Opcode::SLOAD,
-            0x55 => Opcode::SSTORE,
-            0x59 => Opcode::MSIZE,
-            0x5A => Opcode::GAS,
-            0x5E => Opcode::MCOPY,
-            0x5C => Opcode::TLOAD,
-            0x5D => Opcode::TSTORE,
-            0xF0 => Opcode::CREATE,
-            0xF1 => Opcode::CALL,
-            0xF2 => Opcode::CALLCODE,
-            0xF3 => Opcode::RETURN,
-            0xF5 => Opcode::CREATE2,
-            0xF4 => Opcode::DELEGATECALL,
-            0xFA => Opcode::STATICCALL,
-            0xFD => Opcode::REVERT,
-            0xFF => Opcode::SELFDESTRUCT,
-            _ => Opcode::INVALID,
-        }
+        static OPCODE_TABLE: [Opcode; 256] = const {
```

Comment:
Why not `const` rather than `static` for `OPCODE_TABLE`?
Also, don't use `allow`, use `expect`. That way we can know in advance if they're actually allowing something we're not doing.